services/checks/checks.go

```diff
@@ -967,6 +1035,16 @@ func (s *Service) updateChecks(mySQLChecks, postgreSQLChecks, mongoDBChecks []ch
 	s.mongoDBChecks = mongoDBChecks
 }
 
+// updateIntervals updates STT restart timers intervals.
+func (s *Service) updateIntervals(rare, standard, frequent time.Duration) {
```

Comment:
this method seems to be unused right now.

Reply:
Already used)